dags/openshift_nightlies/config/install/azure/chaos.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
these don't even work. Either make real configs or reference existing ones in the manifest.

Reply:
Forgot to double check and replace with config for azure and gcp. My apologies.